CHANGELOG.md

```diff
@@ -2,11 +2,13 @@
 
 All notable changes to this project will be documented in this file, in reverse chronological order by release.
 
-## 3.1.3 - TBD
+## 3.2.0 - TBD
 
 ### Added
 
-- Nothing.
+- [zendframework/zend-di#51](https://github.com/zendframework/zend-di/pull/51) adds `Laminas\Di\GeneratedInjectorDelegator` to decorate the
+  default injector with an AoT generated one.
+- [#11](https://github.com/laminas/laminas-di/pull/11) adds a config option to configure a logger for the AoT generator.
```

Comment:
I have resolved the merge conflicts for this file.
As far as I can tell, the TBD is more of a placeholder and all manual entries get completely replaced by the automatic releases workflow.
Is it necessary to manually maintain the changelog when merging a PR in the future?

Reply:
I recommend manually maintaining the changelog to communicate user-facing changes:

- "There was a bug that had this effect, that is now fixed"
- "We have added this feature; here is a link for how to use it"
- "We are deprecating the following functionality - start using this instead"
- "We removed this specific functionality; you MUST update your code if it uses it"

etc.

For things like documentation changes, QA tooling changes, etc, there's no real need to update the changelog manually.

The "TBD" verbiage _is_ a placeholder... and the tooling looks for it so it knows where to update the date. If it's not found, it essentially skips the whole process of updating the changelog.